network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/about/NetpAboutVpnActivity.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.networkprotection.impl.about
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import com.duckduckgo.anvil.annotations.InjectWith
+import com.duckduckgo.app.global.DuckDuckGoActivity
+import com.duckduckgo.di.scopes.ActivityScope
+import com.duckduckgo.mobile.android.ui.viewbinding.viewBinding
+import com.duckduckgo.networkprotection.impl.databinding.ActivityNetpInfoVpnBinding
+
+@InjectWith(ActivityScope::class)
+class NetpAboutVpnActivity : DuckDuckGoActivity() {
+    private val binding: ActivityNetpInfoVpnBinding by viewBinding()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+        setupToolbar(binding.includeToolbar.toolbar)
+    }
+
+    override fun onSupportNavigateUp(): Boolean {
+        finish()
+        return true
+    }
+
+    companion object {
+
+        internal fun intent(context: Context): Intent {
+            return Intent(context, NetpAboutVpnActivity::class.java)
+        }
```

Comment:
Why not using the `globalActivityStarter`? even if this screen is private to the impl module...for consistency eventually we should all try to avoid the static `intent` pattern